Altis_Life.Altis/stringtable.xml

```diff
@@ -4296,7 +4296,7 @@
             <Polish>Nie ma tego dużo gotówki na ciebie do przechowywania w pojeździe!</Polish>
             <Portuguese>Você não tem que muito dinheiro em você para armazenar no veículo!</Portuguese>
             <Russian>У вас не так много наличных денег на вас, чтобы хранить в автомобиле!</Russian>
-            <German>Sie haben nicht so viel Geld bei Ihnen im Fahrzeug zu speichern!</German>
+            <German>Du hast nicht genug Geld bei dir, um das Fahreug zu lagern!</German>
```

Comment:
`Du hast nicht genug Geld bei dir, um die eingegeben Menge in das Fahrzeug zu lagern!`